app/uk/gov/hmrc/brm/controllers/BirthEventsController.scala

```diff
@@ -56,10 +56,10 @@ trait BirthEventsController extends BRMBaseController {
 
   private def handleInvalidRequest(request : Request[JsValue], errors: Seq[(JsPath, Seq[ValidationError])])(implicit hc : HeaderCarrier) : Future[Result] = {
     countryAuditor.auditCountryInRequest(request.body)
-    info(CLASS_NAME, "post()", s"error parsing request body as [Payload]")
+    info(CLASS_NAME, "post()", s"[handleInvalidRequest] error parsing request body as [Payload]")
```

Comment:
looking at the params for the method, the post should be the function name

Reply:
forgot to resolve this issue - I remember leaving this one for later because of the '[Payload]' part messing with my brain. will consolidate those two into one.